modules/ossm-configuring-jaeger.adoc

```diff
@@ -31,101 +30,148 @@ The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as
 
+If you are deploying {ProductShortName} and Jaeger in a production environment you must change the template to the `production` template, which uses Elasticsearch for Jaeger's storage needs.
 
-.Default Jaeger parameters for Elasticsearch in production
+Elasticsearch is a memory intensive application. The initial set of nodes specified in the default {product-title} installation may not be large enough to support the Elasticsearch cluster.  You should modify the default Elasticsearch configuration to match your use case and the resources you have requested for your {product-title} installation. You can adjust both the CPU and memory limits for each component by modifying the resources block with valid CPU and memory values. Additional nodes must be added to the  cluster if you want to run with the recommended amount (or more) of memory. Ensure that you do not exceed the resources requested for your {product-title} installation.
+
+.Default "production" Jaeger parameters with Elasticsearch
 [source,yaml]
 ----
   apiVersion: maistra.io/v1
   kind: ServiceMeshControlPlane
   spec:
     istio:
       tracing:
-        jaeger:
-          template: production-elasticsearch
-          elasticsearch:
-            nodeCount: 3
-            redundancyPolicy:
-            resources:
-              requests:
-                memory: "16Gi"
-                cpu: "1"
-              limits:
-                memory: "16Gi"
+      enabled: true
+      ingress:
+        enabled: true
+      jaeger:
+        template: production-elasticsearch
+        elasticsearch:
+          nodeCount: 3
+          redundancyPolicy:
+          resources:
+            requests:
+              cpu: "1"
+              memory: "16Gi"
+            limits:
+              cpu: "1"
+              memory: "16Gi"
+
 ----
 
 .Elasticsearch parameters
 [options="header"]
-[cols="l, a, a"]
-|===
-|Parameter |Values |Description
-|nodeCount
-|integer value
-|Number of Elasticsearch nodes
-
-|cpu
-|Specified in units of cores (e.g., 200m, 0.5, 1)
-|Number of central processing units
-
-|memory
-|Specified in units of bytes (e.g., 200Ki, 50Mi, 5Gi)
-|Memory limit
-|===
-
-.Sample configurations
-[options="header"]
-[cols="l, a, a"]
+[cols="l, a, a, a, a"]
 |===
-| Parameter |Proof of Concept | Minimal Deployment
+|Parameter |Description |Values |Default Value |Examples
 
-|Node count
-|1
-|3
+|tracing:
+  enabled
+|This parameter enables/disables tracing in {ProductShortName}. Jaeger is installed by default.
+|`true`/`false`
+|`true`
+|
 
-|Requests CPU
-|500m
+|ingress:
+  enabled
+|This parameter enables/disables ingress for Jaeger.
+|`true`/`false`
+|`true`
+|
+
+|jaeger
+   template
+|This parameter specifies which Jaeger deployment strategy to use.
+|`all-in-one`/`production`
+|`all-in-one`
+|
+
+|elasticsearch:
+  nodeCount
+|Number of Elasticsearch nodes to create.
+|Integer value.
 |1
+|Proof of concept = 1,
+Minimum deployment =3
```

Comment:
is minimum deployment the correct phrase here? Somewhat odd for POC to be less than the minimum.

Reply:
Minimum supported?

Reply:
I went with "minimum-supported", thanks for the suggestion.